R/helper.R

```diff
@@ -42,8 +42,7 @@ is_shiny_ready <- function(url) {
 } # is_shiny_ready
 
 #' @importFrom httpuv randomPort
-#' @importFrom rstudioapi jobRunScript
-#' @importFrom rstudioapi viewer
+#' @importFrom rstudioapi jobRunScript jobGetState viewer
 #' @importFrom fs file_exists
 #' @importFrom withr defer
```

Comment:
make sure we run rcmdcheck before pushing. We may need to do

`#' @importFrom utils packageDescription compareVersion`

Reply:
We don't have to import from utils since it's like base R
Also, each PR runs rcmdcheck as part of the github action

Reply:
I've had rcmdcheck having a warning/note for me before because I didn't have this
https://github.com/A2-ai/ghqc/blob/ade0d77ff31c580d4393b31445a00f0ea1571a40/R/check_packages.R#L48

Reply:
I've been blindly trusting the GHA

<img width="291" alt="Screenshot 2024-12-11 at 3 00 34 PM" src="https://github.com/user-attachments/assets/9c87310f-3a8f-4068-bc2c-cf0b188022ac" />